Source/Wikitivity.CreateInstanceSetting/CreateWebAPIInstanceSetting

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Net;
+using System.Runtime.InteropServices;
+using kCura.EventHandler;
+using kCura.EventHandler.CustomAttributes;
+using kCura.Relativity.Client;
+using Relativity.API;
+using Relativity.Services.Objects;
+
+namespace Wikitivity.CreateInstanceSetting
+{
+	[kCura.EventHandler.CustomAttributes.Description("Post Install EventHandler")]
+	[System.Runtime.InteropServices.Guid("43a79727-0be3-4334-b04d-5579dff70d03")]
+	public class PostInstallEventHandler : kCura.EventHandler.PostInstallEventHandler

```

Comment:
Is this file even needed? I don't think it does anything and looks like it was created from a VS template hence all the scaffolding code.